docs/book/src/quick-start.md

```diff
@@ -22,34 +22,23 @@ Projects created by Kubebuilder contain a Makefile that will install tools at ve
 - [controller-gen](https://github.com/kubernetes-sigs/controller-tools)
 
 The versions which are defined in the `Makefile` and `go.mod` files are the versions tested and therefore is recommend to use the specified versions.
- 
+
 </aside>
 
 ## Installation
 
 Install [kubebuilder](https://sigs.k8s.io/kubebuilder):
 
 ```bash
-os=$(go env GOOS)
-arch=$(go env GOARCH)
-
-# download kubebuilder and extract it to tmp
-curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
-```
-
-If you are using a Kubebuilder plugin version less than version `v3+`, you must configure the Kubernetes binaries required for the [envtest][envtest], run: 
-
-```bash
-# move to a long-term location and put it on your path
-# (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
-sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
-export PATH=$PATH:/usr/local/kubebuilder/bin
+# download kubebuilder and install locally.
+curl -L -o kubebuilder https://go.kubebuilder.io/dl/3.0.0-beta.0/$(go env GOOS)/$(go env GOARCH)
```

Comment:
What about?

```suggestion
curl -L -o kubebuilder https://go.kubebuilder.io/dl/latest/$(go env GOOS)/$(go env GOARCH)
```

Reply:
Using `latest` isn't backwards compatible, since v4.0.0 will likely have the same `latest` endpoint.

Reply:
Makes complete sense. Leave it as it is.

Reply:
I've backtracked and think latest makes the most sense (no updates needed for new releases either). This link can be updated to the last v3.y.z version prior to v4.0.0 when the time comes.